app/tests/test_wallet.py

```diff
@@ -1,9 +1,31 @@
-import routers.governance
+
```

Comment:
even this simple test was very helpful to validate the call was working (after the refactor)